src/main.cc

```diff
@@ -503,7 +503,10 @@ load_cfg(void)
 	 */
 	say_info("%s %s %s", tarantool_package(), tarantool_version(),
 		 BUILD_INFO);
-	say_info("log level %i", cfg_geti("log_level"));
+	if (box_say_log_level() != 0) {
```

Comment:
I don't understand why we need to invoke a Lua function to get the actual log level. Can't we just print the value of the [`log_level`](https://github.com/tarantool/tarantool/blob/6a3e31333d3a5a10f44da6375e8bae6b1c7f88d8/src/lib/core/say.h#L72) variable, which is defined in C?

Reply:
That variable is not very reliable, because:
https://github.com/tarantool/tarantool/blob/6a3e31333d3a5a10f44da6375e8bae6b1c7f88d8/src/lib/core/say.c#L53-L58
That is, currently it can be different from the log_level config option in 2 cases:
1. flightrec log level overrided it;
2. `box.cfg{log_level=1, log_modules={tarantool=2}}`
set log_level variable to 2 (I guess it's ok to print is as `2` in this case).

Reply:
Can we add `say_get_log_level()` function that would return `log_default->level` then? This should be much easier than calling a Lua function from C code.

Reply:
Done.